### PR TITLE
Update assign-licenses-to-users.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated Step6 from selecting the user to double-click the user.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6703

File: microsoft-365/admin/manage/assign-licenses-to-users.md [hunk 1]
```diff
@@ -188,7 +188,7 @@ You can invite guest users to collaborate with your organization in the Azure Ac
 
 5. In the next menu, select **Guest**.
 
-6. In the list of results, select the user who needs a license.
+6. In the list of results, Double Click on the user who needs a license.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/assign-licenses-to-users.md]:
```suggestion
6. In the list of results, double click on the user who needs a license.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Gentle reminder.

**Comment by cmcatee-MSFT** [comment]
@msbemba We don't use the terms "click" or "double-click" because of device and accessibility reasons. Wording is fine as-is.

**Comment by msbemba** [comment]
Thank you for the update @cmcatee-MSFT

**Pull request closed without merging**